packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -65,6 +65,7 @@ export const getCheckboxStyles = stylesFactory((theme: GrafanaTheme2) => {
       position: relative;
       padding-left: ${checkboxSize};
       vertical-align: middle;
+      min-height: ${theme.spacing(3)};
```

Comment:
I think this makes the checkbox higher than it needs to be, not sure what's of the purpose of this?

Reply:
This is just how it was before Torkel's PR the other day. I'm "meeting in the middle" of setting a min-height rather than fixed height. https://github.com/grafana/grafana/pull/35022/files#diff-cd54aedaf6741ed13e25c8581ba73619a994711d9998eb4e0952f4436e2204f4L68

Reply:
As my original more "proper" fix regressed Checkbox worse in places I didn't see, I thought it would be just safer to change it back to how it was originally, rather than a better fix.